.github/workflows/bazel_test.yml

```diff
@@ -27,13 +27,15 @@ jobs:
         run: |
           sudo apt-get install python3-setuptools
           pip3 install wheel absl-py
-      - name: Run all tests
-        working-directory: ./
-        # Build //examples/...
-        # Add "bazel test --config=clang //... --test_tag_filters=test"in the future if any test rules are added
+          alias python='python3'
+      - name: Build examples
         # The -FDP is for filtering out the files which contains "#include <fuzzer/FuzzedDataProvider.h>",
         # The "<fuzzer/FDP...>" is only supported since clang-10 while the clang version in github runner is 9
         run: |
-          alias python='python3'
           bazel build --verbose_failures --build_tag_filters=fuzz_test,-FDP --config=asan-libfuzzer //examples/...
+      - name: Run common tests
+        run: |
+          bazel test --config=clang //... --test_tag_filters=test --build_tag_filters=test
```

Comment:
Why are the `--test_tag_filters` and `--build_tag_filters` flags necessary here?

Reply:
The scenario in my mind is that in the future we may need the CI to run more tests like this. If we don't use the tag filters, instead we test the specific target here, then this file need to update every time a new test is added.

It is my assumption that the benefit getting from no need to update this CI script frequently is more than the potential risk that we may forget to add the tag. If my assumption is not correct or there is a way to achieve the goal without the tag filters, I absolutely support to delete the filters.

Reply:
Used `--build_tests_only` instead.